setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Download and prepare raw data
+mkdir data
+cd data
+wget https://zenodo.org/record/5101839/files/epic6.tar.gz
+tar -xvf epic6.tar.gz
```

Comment:
The "-v" flag is for verbose. It prints everything to the terminal. It may be removed.